unittests/api_tests.cpp

```diff
@@ -297,17 +297,17 @@ struct MySink : public bio::sink
 };
 uint32_t last_fnc_err = 0;
 
-BOOST_FIXTURE_TEST_CASE(action_receipt_tests, validating_tester) { try {
-   produce_blocks(2);
-   create_account( "test"_n );
-   set_code( "test"_n, test_contracts::payloadless_wasm() );
-   produce_blocks(1);
+BOOST_FIXTURE_TEST_CASE_TEMPLATE(action_receipt_tests, T, validating_testers, T) { try {
+   this->produce_blocks(2);
```

Comment:
Personally, I prefer the approach used in the prior PR:
```
BOOST_AUTO_TEST_CASE_TEMPLATE( missing_sigs, TESTER, validating_testers ) { try {
   TESTER chain;
```
Over using the less documented `BOOST_FIXTURE_TEST_CASE_TEMPLATE` and having `this->`. To me `chain.` is better than `this->`. 

I'm also concerned about `BOOST_AUTO_TEST_CASE_TEMPLATE` not being maintained by Boost.